data/map systems.txt

```diff
@@ -22077,6 +23314,53 @@ system "Mei Yohn"
 			distance 312
 			period 29.0088
 
+system Meissa
+	pos -208.47222 156.86111
+	government Syndicate
+	link "Al Dhanab"
+	link Alpheratz
+	link Atlas
+	asteroids "small rock" 6 5.082
+	asteroids "medium rock" 8 6.6
+	asteroids "large rock" 13 4.224
+	asteroids "small metal" 9 5.28
+	asteroids "medium metal" 10 6.27
+	asteroids "large metal" 15 3.894
+	minables aluminum 1 7.0420918
+	minables tungsten 12 6.825799
+	minables uranium 1 9.7583272
+	haze _menu/haze-133
+	fleet "Large Core Pirates" 800
+	fleet "Small Core Pirates" 1200
+	fleet "Small Core Merchants" 600
```

Comment:
```suggestion
	fleet "Small Core Pirates" 3000
	fleet "Small Core Merchants" 2000
	fleet "Large Core Merchants" 4000
```
Pirates are unlikely to get here through Atlas with any great frequency, Syndicate forces probably aren't going to bother patrolling this system, so most of the fleets here would just be merchants either lost or avoiding Atlas.